encoders/numeric/UMAPEncoder/README.md

```diff
@@ -0,0 +1,3 @@
+# UMAPEncoder
+
+`UMAPEncoder` encodes data from an ndarray of size `B x T` into an ndarray of size `B x D` using _Uniform Manifold Approximation and Projection (UMAP)_ for dimensionality reduction. Full code and documentation can be found here: [https://github.com/lmcinnes/umap]
```

Comment:
Could you follow this template for `README`:
https://github.com/jina-ai/jina-hub/pull/5782/files
especially for the param initialisation.